medcat/utils/relation_extraction/rel_dataset.py

```diff
@@ -28,15 +27,16 @@ def __init__(self, tokenizer: TokenizerWrapperBERT, config: ConfigRelCAT, cdb: C
             interest are surrounded by the special tokens, see create_base_relations_from_csv doc.
 
         Args:
-            tokenizer (TokenizerWrapperBERT): okenizer used to generate token ids from input text
+            tokenizer (TokenizerWrapperBERT | TokenizerWrapperLlama | TokenizerWrapperModernBERT): 
+                tokenizer used to generate token ids from input text
             config (ConfigRelCAT): same config used in RelCAT
             cdb (CDB): Optional, used to add concept ids and types to detected ents, 
                 useful when creating datasets from MedCAT output. Defaults to CDB().
         """
 
         self.cdb: CDB = cdb
         self.config: ConfigRelCAT = config
-        self.tokenizer: TokenizerWrapperBERT = tokenizer
+        self.tokenizer: Union[TokenizerWrapperBERT, TokenizerWrapperLlama, TokenizerWrapperModernBERT] = tokenizer
```

Comment:
Again, protocol would help - see comment in line 96 in `rel_cat.RelCAT.__init__`